pkg/jx/cmd/get_apps.go

```
@@ -93,21 +129,58 @@ func (o *GetAppsOptions) Run() error {
 		return nil
 	}
 
-	table := o.generateTable(apps, kubeClient)
+	if o.ShowStatus {
+		if checkDifferentAppsInAppsSlice(apps) {
+			fmt.Fprint(o.Out, "Different apps provided. Provide only one app to check the status of\n")
+			return nil
+		}
+		return o.generateAppStatusOutput(&apps.Items[0])
+	}
 
+	if o.Output != "" {
+		appsResult := o.generateTableFormatted(apps)
+		return o.renderResult(appsResult, o.Output)
+	}
+	table := o.generateTable(apps, kubeClient)
 	table.Render()
 	return nil
 }
 
+func (o GetAppsOptions) generateAppStatusOutput(app *v1.App) error {
+	output, _ := o.Helm().StatusReleaseWithOutput(o.Namespace, app.Labels[helm.LabelAppName], "json")
```

I wouldn't ignore the error here. Maybe print a warning message.